topi/python/topi/util.py

```diff
@@ -213,3 +214,16 @@ def select_array(i, j):
         return now
 
     return tvm.compute(matrix.shape, select_array, name=name)
+
+
+@tvm.register_func("print_tensor")
```

Comment:
this is not used, remove for now

Reply:
I found it is quite useful when debugging. Do we have any debug option to print intermediate result?

Reply:
sure, maybe we can add as an util later as separate PR, but we need documents on these